src/content/en/updates/2017/09/autoplay-policy-changes.md

```diff
@@ -0,0 +1,175 @@
+  allow autoplay with sound.
+
+### Media Engagement Index (MEI) {: #mei }
+
+The MEI measures an individual's propensity to consume media on a site.
+Chrome's current approach is a ratio of visits to significant media playback
+events per origin:
+
+- Consumption of the media (audio/video) must be greater than 7 seconds.
+- Audio must be present and unmuted.
+- Tab with video is active.
+- Size of the video (in px) must be greater than [200x140].
+
+From that, Chrome calculates a Media Engagement score which is highest on sites
+where media is played on a regular basis. When it is high enough, media playback
+is allowed to autoplay on desktop only.
+
+Note that you can check out user's MEI at the <i>chrome://media-engagement</i>
+internal page in Chrome 62.
+
+<figure>
+  <img src="/web/updates/images/2017/09/media-engagement.png"
+       alt="Screenshot of the chrome://media-engagement page">
+  <figcaption>
+    <b>Figure 2.</b>
+    Screenshot of the <i>chrome://media-engagement</i> internal page
+  </figcaption>
+</figure>
+
+### Iframe delegation {: #iframe }
+
+Once an origin has received autoplay permission, it can delegate that
+permission to iframes via a new HTML attribute. Check out the [Gesture
+Delegation API proposal] to learn more.
+
+    <iframe src="myvideo.html" gesture="media">
+
+Without iframe delegation, videos will not be able to autoplay with sound.
+
+### Example scenarios
+
+<b>Example 1:</b> Every time a user visits <i>VideoSubscriptionSite.com</i> on their
+laptop they watch a TV show or a movie. As their Media Engagement score is
+high, autoplay is allowed.
+
+<b>Example 2:</b> <i>GlobalNewsSite.com</i> has both text and video content.
+Most users go to the site for text content and watch videos only occasionally.
+Users' Media Engagement score is low, so autoplay wouldn't be allowed if a user
+navigates directly from a social media page or search. 
+
+<b>Example 3:</b> <i>LocalNewsSite.com</i> has both text and video content.
+Most people enter the site through the homepage and then click on the news
+articles. Autoplay on the news article pages would be allowed because of user
+interaction with the domain. However, care should be taken to make sure users
+aren't surprised by autoplaying content.
+
+<b>Example 4:</b> <i>MyMovieReviewBlog.com</i> embeds an iframe with a movie
+trailer to go along with their review. The user interacted with the domain to
+get to the specific blog, so autoplay is allowed. However, the blog needs to
+explicitly delegate that privilege to the iframe in order for the content to
+autoplay.
+
+## Best Practises for Web Developers {: #best-practises }
+
+Here's the one thing to remember: Don't ever assume a video will play, and
+don't show a pause button from the start. It is so important that I'm going
+to write it one more time below for those who simply skim through that post.
+
+Key Point: Don't ever assume a video will play, and don't show a pause button
+from the start.
```

Comment:
It might be better to say don't show a pause button when the video's not actually playing. (It took me a few seconds to realize what you were getting at with your pause remark.)

Reply:
Thanks. Fixed.